styles/components/_buttons.scss

```diff
@@ -21,12 +21,13 @@
   &:hover {
     text-decoration: none;
     cursor: pointer;
-    border: solid 1px $focusedBorder;
+    border-color: $focusedBorder;
   }
 
   &:focus {
     text-decoration: none;
     outline: 0;
+    border-color: $focusedBorder;
```

Comment:
The border should change on focus too, for anyone using a keyboard to tab to the buttons.